### PR TITLE
Feature/jitsi

### PR DESCRIPTION
**Pull request by AAfify1**
Jitsi Implementation

### CODE REVIEW

File: TableSDK/Table SDK/SDK/ViewControllers/JitsiVideo/JitsiVideoVC.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    @IBOutlet var jitsiMeetView: JitsiMeetView!
+    var tenant = ""
+    var roomID = ""
+    var token = ""
+    var userInfo: UserAttributes = UserAttributes()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.navigationBar.isHidden = true
+        let tokenizedURL = "https://8x8.vc/" + self.tenant + "/" + self.roomID + "?jwt=" + self.token
```
Comment (gazreese) [inline TableSDK/Table SDK/SDK/ViewControllers/JitsiVideo/JitsiVideoVC.swift:23]: Is this URL going to be the same with dev / production? Do we need to double check with the Table team?

**Review by gazreese — CHANGES_REQUESTED** [review]
Code looks great, but just need to address these before we squash and merge into master.

Might be worth copying-over the sample app as-is to the private repo so that we've got a way to get back to a testable version?

File: TableSDK/Podfile [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
 
   # Pods for Table SDK
+  pod 'JitsiMeetSDK'
   pod 'OpenTok'
+ 
+
+
+end
 
+post_install do |installer|
```
[the remark this answers is not in the document]
Comment (AAfify1) [reply TableSDK/Podfile:18]: This was required for Jitsi in their documentation

File: SampleProject/TableSample/ViewController.swift [hunk 3]
```diff
@@ -14,7 +14,7 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
       
-        Table.initialize(workspaceUrl: "https://YOUR_WORKSPACE.table.co", apiKey: "YOUR_SDK_API_KEY", onSuccessInitializeCompletion: {
+        Table.initialize(workspaceUrl: "https://develop3.dev.table.co", apiKey: "test_key", onSuccessInitializeCompletion: {
```
[the remark this answers is not in the document]
Comment (AAfify1) [reply SampleProject/TableSample/ViewController.swift]: changed and pushed

File: SampleProject/TableSample/ViewController.swift [hunk 4]
```diff
@@ -23,8 +23,8 @@ class ViewController: UIViewController {
 
     @IBAction func registerUser(_ sender: Any) {
         var userAttrib = UserAttributes()
-        userAttrib.firstName = "User"
-        userAttrib.lastName = "Name"
+        userAttrib.firstName = "IOS"
```
[the remark this answers is not in the document]
Comment (AAfify1) [reply SampleProject/TableSample/ViewController.swift]: changed and pushed

**Review by gazreese — APPROVED** [review]

**Review by gazreese — APPROVED** [review]

**Pull request merged**